docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -215,8 +215,37 @@ Imports
 Template style
 ==============
 
-* In Django template code, put one (and only one) space between the curly
-  brackets and the tag contents.
+* In Django template code: 
+
+  The ``{% extends %}`` should be the first non-comment line
+
+  Do this:
+
+  .. code-block:: html+django
+
+      {% extends %}
+
+      Other code goes here
+
+  Or this:
+
+  .. code-block:: html+django
+
+      # This is a comment
+      {% extends %}
+
+      Other code goes here
+
+  Don't do this:
+
+  .. code-block:: html+django
+
+      {% load foo %}
```

Comment:
Let’s avoid using “foo” / “bar” since they’re meaningless and unrealistic examples that can confuse new people. I know the existing example did, but we can change that.

For example, here, we can name a real template library:

```suggestion
      {% load i18n %}
```

Below we could use `{% load humanize i18n %}`, and variables like `{{ user }}` , `{{ user.name|lower }}`, `{% block header %}`, etc.